src/opensprinkler-card.ts

```diff
@@ -209,6 +209,11 @@ export class OpensprinklerCard extends LitElement {
   private _secondaryText() {
     const entities: EntitiesFunc = p => this._matchingEntities(p)
 
+    if (hasRainDelayActive(entities)) {
+      const stop_time = entities(isRainDelayStopTime).find(_ => true)?.state;
+      return `Rain delay${ stop_time_hacked ? ` ends ${relativeTime(new Date(stop_time), this.hass!.locale)}` : ''}`;
```

Comment:
Bah. Straight up bug - I have a local commit that hacks the time offset until the fixed HACS integration is deployed.

Reply:
Fixed in ee9bfc5